package.json

```diff
@@ -869,6 +869,7 @@
         "ms-rest-azure": "^2.3.1",
         "pg": "^7.18.1",
         "pg-structure": "^5.10.7",
+        "public-ip": "^4.0.0",
```

Comment:
I wasn't sure if adding this dependency was the best way to go about finding the machine's IP. Does anyone know of a better way?

Reply:
How about this package, which has zero dependencies and way more weekly downloads (the main two things I look at when picking something to use): https://www.npmjs.com/package/ip

Reply:
I looked into that package and a few others but they all gave the private IP instead of the public one. `public-ip` was the only package I could find that provides a public ip

Reply:
Interesting distinction - did you by chance find any best practice docs around firewalls for Postgres? I'm wondering how "recommended" it is to do this kind of stuff... as far as I understand it the ip address that you're talking about could easily change based on the whim of your internet service provider

Reply:
I was going off of how the portal configures the client IP address:

<img width="1386" alt="Screen Shot 2020-03-31 at 12 55 11 PM" src="https://user-images.githubusercontent.com/22795803/78069631-470f0a00-734f-11ea-9fd6-8aea35d5e772.png">

That being said, the address could change. I'm not sure if it would be better to update the existing firewall rule in that case or create a new one. Right now we just update it